openshift-ci/Dockerfile.registry.build.master

```diff
@@ -0,0 +1,24 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS stage
+
+COPY . /ocs-operator
+
+WORKDIR /ocs-operator
+
+RUN microdnf install -y jq
+RUN hack/generate-master-appregistry.sh
+
+
+FROM quay.io/openshift/origin-operator-registry:latest
+
+COPY --from=stage /ocs-operator/build/_output/appregistry/olm-catalog /registry/ocs-catalog
+
+# replaces ocs-operator image with the one built by openshift ci
+RUN find /registry/ocs-catalog/ -type f -exec sed -i "s|image\: .*/ocs-operator:.*$|image: default-route-openshift-image-registry.apps.build02.ci.devcluster.openshift.com/${OPENSHIFT_BUILD_NAMESPACE}/stable:ocs-operator|g" {} \; || :
```

Comment:
A note on this: The CI team would REALLY prefer we don't do this, but I don't think they've managed to come up with a way to not require us to use the hardcoded URL. Their preference would probably for us to not build an appregistry container anymore and just use bundles, since they have worked out automation for building index images of bundles specified in the repo. I'm fine with merging this for now, but if we can find a way to do without it later that would be ideal.